debian/node-red-contrib-revpi-nodes/update-node-red-revpi-nodes

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: GPL-2.0
+# Copyright: 2023 KUNBUS GmbH
+#
+# This script updates the node-red-contrib-revpi-nodes, which must always be
+# installed for the appropriate server version.
+
+# As long as the node-red is installed in /home/pi/.node-red, we have to
+# upgrade the nodes in that location as user pi.
+
+NODE_RED_ROOT="/home/pi/.node-red"
+REVPI_NODES_VERSION="1.1.0"
+
+# Check node red installation directory
+if [ ! -d "${NODE_RED_ROOT}" ]; then
+    echo "NodeRED is not installed at '${NODE_RED_ROOT}'"
+    exit 1
+fi
+
+# Check user is root or pi, both can upgrade the nodes
+if [ "$(id -un)" == "pi" ]; then
+    # This will install/upgrade the revpi nodes of this package as pi user
+    /usr/bin/npm install \
+        --save --save-exact \
```

Comment:
`--save` is the default and its not necessary to provide it 
although imho it does not hurt here to make clear that this save to the package.json in the target dir